templates/components/element_detail_tabs.html

```diff
@@ -0,0 +1,458 @@
+                  <button type="button" name="save" value="save" class="btn btn-xs btn-success" onclick="save_smt('smt_{{ forloop.counter }}');return false;">Save</button>
+                </div>
+
+                </form>
+              </div>
+
+                </div>
+              </div>
+            </div>
+
+            {% endfor %}
+          </div><!--/smt-list-->
+          <div><h3><button type="submit" id="new_component_statement" class="small btn btn-md btn-success" style="color: white;" onclick="add_smt()">Add component statement</button></h3></div>
+</div>
+
+      <!-- Tab panel: combined -->
+      {% if enable_experimental_oscal %}
+      <div role="tabpanel" class="tab-pane" id="oscal">
+        <p id="oscal_download" class="navbar-text navbar-right">
+          <a id="oscal_download_json_link"
+             class="navbar-link" 
+             href="/systems/{{ system.id }}/component/{{ element.id }}/download/oscal/json">
+            <span class="glyphicon glyphicon-download"></span>
+            Download ...
+          </a>
+        </p>
+        <div id="combined_smt" class="control-text"><h3>OSCAL (under development)</h3></div>
+        <div style="font-family: monospace; font-size:12px; white-space: pre;">{{ oscal }}</div>
+      </div>
+      {% endif %}
+
+      <!-- Tab panel: opencontrol -->
+      {% if enable_experimental_opencontrol %}
+      <div role="tabpanel" class="tab-pane" id="opencontrol">
+        <div id="combined_smt" class="control-text"><h3>OpenControl (under development)</h3></div>
+        <div style="font-family: monospace; font-size:12px; white-space: pre;">{{ opencontrol }}</div>
+      </div>
+      {% endif %}
+
+    </div>
+  </div>
+
+
+  </div>
+</div>
+
+{{ block.super }}
+{% endblock %}
+
+{% block scripts %}
+      <script>
+        // View combined implementation statement
+
+        // Adds statement form to page
+        function add_smt() {
+          var panel_num = $('.panel').length + 1
+          var smt_form = `
+            <div id="panel-panel_num" class="panel panel-default">
+            <div class="panel-heading" role="tabx" id="document-panel_num-title">
+              <h4 id="panel-panel_num-title" class="panel-title">
+                <a role="button" data-toggle="collapse" data-parent="#accordion" href="#document-panel_num-body" aria-expanded="{% if forloop.first %}true{% endif %}" aria-controls="document-panel_num-body">
+                  <span id="producer_element-panel_num-title">{{ element.name }}</span>
+                  <div class="panel-heading-smt"></div>
+                </a>
+              </h4>
+            </div>
+            <div id="document-panel_num-body" class="panel-collapse collapse {% if forloop.first %}in{% endif %} output-document" role="tabpanel" aria-labelledby="document-panel_num-title">
+              <div class="panel-body output-document">
+
+            <div>
```

Comment:
Is the double-div necessary/intended?

Reply:
@omarabed15 Divs are correct. Indentations are not clear. I have significantly refactored indentations in page to make code folding and div analysis easier.